src/containers/TransactionsListItem/TransactionsListItem.css

```diff
@@ -26,3 +26,10 @@
     background-color: brown;
     margin: 10px 10px 10px 0;
 }
+.transactions-list-item__image{
+    height: auto;
+    min-width: 20px;
+    max-height: 100%;
```

Comment:
`max-height: 100%` means that image will be the same height as container, it's better to set it to 20px to keep square-ness

Reply:
I'm ready to merge it as soon as you change max-height :)

Reply:
Done.